src/umpire/ResourceManager.hpp

```diff
@@ -317,6 +317,7 @@ class ResourceManager {
   int getNextId() noexcept;
 
   std::string getAllocatorInformation() const noexcept;
+  std::string getResourceInformation() const noexcept;
```

Comment:
Where is this function implemented?

Reply:
good question!  I have no idea.  I must have inadvertantly picked it up during a merge from somewhere?  I honestly don't remember adding it, but `git blame` doesn't lie.  I'll remove.

Reply:
Fixed.